rules/Php80/Rector/Class_/ClassPropertyAssignToConstructorPromotionRector.php

```diff
@@ -161,4 +158,21 @@ private function decorateParamWithPropertyPhpDocInfo(Property $property, Param $
         $paramType = $this->staticTypeMapper->mapPhpParserNodePHPStanType($param->type);
         $this->varTagRemover->removeVarPhpTagValueNodeIfNotComment($param, $paramType);
     }
+
+    private function shouldSkipParam(Param $param): bool
+    {
+        if ($param->variadic) {
+            return true;
+        }
+
+        if ($param->type instanceof NullableType && $this->isName($param->type->type, 'callable')) {
+            return true;
+        }
+
+        if ($param->type instanceof Identifier && $this->isName($param->type, 'callable')) {
+            return true;
+        }
```

Comment:
should can be simplified with: 

```suggestion
        $type = $param->type instanceof NullableType
            ? $param->type->type
            : $param->type;

        if ($type instanceof Identifier && $this->isName($type, 'callable')) {
            return true;
        }
```

Reply:
done